playbooks/common/openshift-cluster/upgrades/pre/verify_cluster.yml

```diff
@@ -41,6 +41,13 @@
     - openshift_release is defined
     - not (openshift_release is version_compare(openshift_upgrade_target ,'='))
 
+  - fail:
```

Comment:
I thought we were forcing masters to be schedulable?

Would it make more sense to do this during init/facts ?  Don't we need the masters to be schedulable during installs too?

Reply:
Some masters might be left unschedulable. During the upgrade these would be forcibly made schedulable, upgraded and then `openshift_schedulable` would be applied again in `openshift_manage_nodes`.

If the inventory has the same settings as in 3.7 webconsole would fail to be installed, so this error would make admin fix the inventory, giving them an option to leave some masters disabled